### DISCUSSION TITLE
vip下载

### DISCUSSION THREAD

**Discussion by yangwenjie1231** [Beadd/MusicDownloader#70] [Ideas]
这个网站有vip解析，可以抓取这个网站的api
https://tools.qzxdp.cn/wyy_vip#:~:text=网易云VIP音乐[https://tools.qzxdp.cn/wyy_vip#:~:text=网易云VIP音乐](url)

**Comment by Beadd** [comment]
> 这个网站有vip解析，可以抓取这个网站的api [https://tools.qzxdp.cn/wyy_vip#:~:text=网易云VIP音乐[https://tools.qzxdp.cn/wyy_vip#:~:text=网易云VIP音乐](url)](https://tools.qzxdp.cn/wyy_vip#:~:text=%E7%BD%91%E6%98%93%E4%BA%91VIP%E9%9F%B3%E4%B9%90%5Bhttps://tools.qzxdp.cn/wyy_vip#:~:text=%E7%BD%91%E6%98%93%E4%BA%91VIP%E9%9F%B3%E4%B9%90%5D(url))

我试了一下，但这个使用2次后就有限制了

**Comment by Beadd** [comment]
已更新，[v5.0.0](https://github.com/Beadd/Creamplayer/releases/tag/v5.0.0)
